app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt

```diff
@@ -128,6 +277,22 @@ class ProfileFragment : Fragment(), ProfileView, ActionMode.Callback {
     }
 
     private fun listenToChanges() {
+        //setup rx for avatar
+        imageObservable.subscribe({ _ ->
+            if (isAvatarChanged) {
+                startActionMode()
+            } else {
+                finishActionMode()
+            }
+        })
+
+        Observables.combineLatest(text_name.asObservable(), text_username.asObservable(), text_email.asObservable()).subscribe({ t ->
```

Comment:
Why are you adding another ` Observables.combineLatest(..)` here? You can user the other one.